src-java/floodlight-service/floodlight-modules/README.md

```diff
@@ -15,3 +15,45 @@ where:
  ratePkts - is the rate in packets for specific type of rule
  disco_packet_size - the size of discovery packet, is being defined in properties file with the key 'org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size'.
 ```
+
+#### How to debug Floodlight component
+
+In order to run floodLight in the debug mode we have to add this line of arguments
```

Comment:
the authors of Floodlight write Floodlight with only the first capital letter. You need a comma after the phrase `in order to` ends: In order to ... **,** we have to ...